### PR TITLE
Package shipping scrpting examples

### PR DESCRIPTION
**Pull request by waljo11**
**Description of changes**

### CODE REVIEW

**Comment by waljo11** [comment]
Endevor Package shipping example and alternative models and skeletons

File: endevor/Package-Shipping-Model-and-Script-Examples/ISPF variables.txt [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+VDDHSPFX V   PUBLIC.HOST   
+VDDRSPFX V   PUBLIC.RMOT   
+
+VNBSQDSP     SHIP PAC 'FINA#YEDN0607479' TO DEST SOMWHER OPT OUT PREF 
+
+VDDDSTID V   SOMWHER  
+VNBLSDST V   SOMWHER  
+
+
+VDDCUSER V   WALJO11
```
Comment (adam-wolfe) [inline endevor/Package-Shipping-Model-and-Script-Examples/ISPF variables.txt:30]: Replace user ID

File: endevor/Package-Shipping-Model-and-Script-Examples/ISPF variables.txt [hunk 2]
```diff
@@ -0,0 +1,62 @@
...
+VPHXUID  V   WALJO11   
+
+VDDXMETH V   XCOM 
+
+
+VDDHSPFX V   PUBLIC.HOST      
+VDDRSPFX V   PUBLIC.RMOT      
+VDDHSPFX V   PUBLIC.LOCL 
+
+VDDXNODE V   10.175.81.10
```
Comment (adam-wolfe) [inline endevor/Package-Shipping-Model-and-Script-Examples/ISPF variables.txt:45]: Replace with generic IP address

File: endevor/Package-Shipping-Model-and-Script-Examples/C1BMXIN.skl [hunk 3]
```diff
@@ -0,0 +1,140 @@
...
+//BSTERR   DD SYSOUT=*
+//*--------------------------------------------------------------------
+//SHOWME   EXEC PGM=IEBGENER,                              C1BMXIN
+//         COND=(4,LT)
+//SYSPRINT  DD SYSOUT=*                           MESSAGES
+//SYSUT1   DD  DSN=&&CSVFILE,DISP=(OLD,PASS)
+//SYSUT2    DD SYSOUT=*                           OUTPUT FILE
+//SYSIN    DD DUMMY                               CONTROL STATEMENTS
+//SYSUDUMP DD SYSOUT=*
+//*----------------------- WALJO11.CLIST.TSO06(ENBPIU00) --------------
```
Comment (adam-wolfe) [inline endevor/Package-Shipping-Model-and-Script-Examples/C1BMXIN.skl:105]: Replace user ID

**Review by adam-wolfe — CHANGES_REQUESTED** [review]
Hi @waljo11, PR looks mostly good, I noted a few items that should be replaced.

**Review by adam-wolfe — APPROVED** [review]
Looks good. Thanks @waljo11.

**Pull request merged**